src/code/audio_playback.c

```diff
@@ -529,10 +529,18 @@ void Audio_SeqLayerNoteRelease(SequenceLayer* layer) {
     Audio_SeqLayerDecayRelease(layer, ADSR_STATE_RELEASE);
 }
 
+/**
+ * Extract the synthetic wave to use from gWaveSamples and update corresponding frequencies
+ *
+ * @param note
+ * @param layer
+ * @param waveId the index of the type of synthetic wave to use, offset by 128
+ * @return s32 harmonicIndex, the log of the harmonic for the synthetic wave generated by gWaveSamples
```

Comment:
I don't think we generally include the return type
```suggestion
 * @return harmonicIndex, the log of the harmonic for the synthetic wave generated by gWaveSamples
```